mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxPathDAOTest.java

```diff
@@ -27,26 +27,29 @@
 import org.apache.james.backends.cassandra.CassandraClusterExtension;
 import org.apache.james.backends.cassandra.components.CassandraModule;
 import org.apache.james.backends.cassandra.versions.CassandraSchemaVersionModule;
+import org.apache.james.mailbox.cassandra.host.CassandraRestartExtension;
 import org.apache.james.mailbox.cassandra.ids.CassandraId;
 import org.apache.james.mailbox.cassandra.modules.CassandraMailboxModule;
 import org.apache.james.mailbox.model.MailboxPath;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
 import nl.jqno.equalsverifier.EqualsVerifier;
 
+@ExtendWith(CassandraRestartExtension.class)
 public abstract class CassandraMailboxPathDAOTest {
-    protected static final String USER = "user";
-    protected static final String OTHER_USER = "other";
-    protected static final CassandraId INBOX_ID = CassandraId.timeBased();
-    protected static final CassandraId OUTBOX_ID = CassandraId.timeBased();
-    protected static final CassandraId otherMailboxId = CassandraId.timeBased();
-
-    public static final MailboxPath USER_INBOX_MAILBOXPATH = MailboxPath.forUser(USER, "INBOX");
-    public static final CassandraIdAndPath INBOX_ID_AND_PATH = new CassandraIdAndPath(INBOX_ID, USER_INBOX_MAILBOXPATH);
-    public static final MailboxPath USER_OUTBOX_MAILBOXPATH = MailboxPath.forUser(USER, "OUTBOX");
-    public static final MailboxPath OTHER_USER_MAILBOXPATH = MailboxPath.forUser(OTHER_USER, "INBOX");
+    private static final String USER = "user";
+    private static final String OTHER_USER = "other";
+
+    static final CassandraId INBOX_ID = CassandraId.timeBased();
+    static final CassandraId OUTBOX_ID = CassandraId.timeBased();
+    static final CassandraId otherMailboxId = CassandraId.timeBased();
+    static final MailboxPath USER_INBOX_MAILBOXPATH = MailboxPath.forUser(USER, "INBOX");
+    static final MailboxPath USER_OUTBOX_MAILBOXPATH = MailboxPath.forUser(USER, "OUTBOX");
+    static final MailboxPath OTHER_USER_MAILBOXPATH = MailboxPath.forUser(OTHER_USER, "INBOX");
+    static final CassandraIdAndPath INBOX_ID_AND_PATH = new CassandraIdAndPath(INBOX_ID, USER_INBOX_MAILBOXPATH);
```

Comment:
please avoid unrelated changes, especially on big commits

Reply:
Ok, I thought it was ok to change the code to fix a bit more but I understand and will try to revert those. Thanks for feedback !

Reply:
no problem to fix a little more, just do it in a separated commit